tests/unit/stubs/index.js

```diff
@@ -4,5 +4,8 @@ export default {
   'oc-icon': true,
   'oc-checkbox': true,
   'oc-button': true,
-  'oc-breadcrumb': true
+  'oc-breadcrumb': true,
+  'oc-drop': true,
+  translate: true,
```

Comment:
```suggestion
  'translate': true,
```

Is this supposed to be quoted?

Reply:
intelliJ complains if I quote it
![image](https://user-images.githubusercontent.com/2425577/121146302-83959a00-c85f-11eb-99e1-d5f49cba562c.png)

Reply:
which is fine. quotes are required when a key contains e.g. a `-`. By default we don't want the quotes.